### PR TITLE
Fix nix flake

### PR DESCRIPTION
**Pull request by Avi-D-coder**
The build was broken, and now it's not.

### CODE REVIEW

**Review by Ailrun — APPROVED** [review]
LGTM

File: flake.nix [hunk 1]
```diff
@@ -72,7 +72,7 @@
             with haskell.lib; {
 
               hls-tactics-plugin = hsuper.hls-tactics-plugin.override {
-                refinery = hself.refinery_0_3_0_0;
+                refinery = hself.refinery;
```
Comment (berberman) [inline flake.nix:75]:
```suggestion

```

File: flake.nix [hunk 2]
```diff
@@ -83,6 +83,37 @@
...
+
+              ghc-source-gen = hself.callCabal2nix "ghc-source-gen"
+                (builtins.fetchTarball {
+                  url =
+                    "https://hackage.haskell.org/package/ghc-source-gen-0.4.1.0/ghc-source-gen-0.4.1.0.tar.gz";
+                  sha256 =
+                    "0kk599vk54ckikpxkzwrbx7z5x0xr20hr179rldmnlb34bf9mpnk";
+                }) { };
+
+              refinery = hself.callCabal2nix "refinery"
```
Comment (berberman) [inline flake.nix:110]: IIRC, we have refinery 0.4.0 in nixpkgs, so maybe we don't need this source override.

Comment (Avi-D-coder) [reply flake.nix]: Good catch, thanks

**Review by berberman — APPROVED** [review]
LGTM, thanks!

**Pull request merged**